tidb-cloud/integrate-tidbcloud-with-zapier.md

```diff
@@ -0,0 +1,250 @@
+---
+title: Integrate TiDB Cloud with Zapier
+summary: Learn how to connect TiDB Cloud to 5000+ Apps with Zapier.
+---
+
+# Integrate TiDB Cloud with Zapier
+
+[Zapier](https://zapier.com/app/dashboard) is an automation tool that lets you easily create workflows that involve thousands of apps and services.
+
+Use TiDB Cloud App on Zapier enables you:
+
+- Use TiDB, a MySQl-compatible HTAP database for free. No need to build locally.
+- Make it easier to manage your TiDB Cloud.
+- Connect TiDB Cloud to 5000+ Apps and automate your workflows.
+
+This guide will give a high-level introduction to TiDB Cloud App on Zapier and an example on how to use it.
+
+# Quick start with template
+
+In this section, we will use the preset template as an example to try out TiDB Cloud App on Zapier.
+
+## Prepare
+
+Before you start, you need:
+
+- A [Zapier account](https://zapier.com/app/login).
+- A [GitHub account](https://github.com/login).
+- A [TiDB Cloud account](https://tidbcloud.com/signup) and a Serverless Tier on TiDB Cloud, See [TiDB Cloud Quick Start](https://docs.pingcap.com/tidbcloud/tidb-cloud-quickstart#step-1-create-a-tidb-cluster) for more details.
+
+## Step 1: Get the template
+
+Go to [TiDB Cloud App on Zapier](https://zapier.com/apps/tidb-cloud/integrations) to choose a template and click **Try it**. Then you will enter the editor page. 
+
+Here we use `Add Github global event into TiDB without custom schema` template as an example. It will add a new row into TiDB Cloud cluster when a new global event is created on GitHub.
```

Comment:
I couldn't find this `Add Github global event into TiDB without custom schema` template on Zapier.

Reply:
The name has been changed. I will change the doc too.